src/api/organization/routes/organization.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * organization router
+ */
+
+import { factories } from '@strapi/strapi';
+
+export default factories.createCoreRouter('api::organization.organization', {
+  config: {
+    find: {
+      auth: false,
+      policies: [],
+      middlewares: [],
+    },
+    findOne: {
+      auth: false,
+      policies: [],
+      middlewares: [],
+    },
+    create: {
+      auth: false,
+      policies: [],
+      middlewares: [],
+    },
+    update: {
+      auth: false,
+      policies: [],
+      middlewares: [],
+    },
+    delete: {
+      auth: false,
+      policies: [],
+      middlewares: [],
+    },
+  },
+});
```

Comment:
```suggestion
export default factories.createCoreRouter('api::organization.organization');
```
生成 API 的权限在后台设置，尽量不在代码强制改变：https://docs.strapi.io/cms/features/users-permissions#editing-a-role